src/components/molecules/VenuePartygoers/VenuePartygoers.tsx

```diff
@@ -28,11 +30,11 @@ export const VenuePartygoers = () => {
 
     return `${numberOfRecentWorldUsers} ${attendeesTitle} online`;
   }, [
-    isCurrentVenueLoaded,
-    parentVenue,
-    currentVenue,
-    numberOfRecentWorldUsers,
+    isLoading,
     isRecentWorldUsersLoaded,
+    parentVenue?.attendeesTitle,
+    currentVenue?.attendeesTitle,
+    numberOfRecentWorldUsers,
```

Comment:
[OOS] I think it should only show sovereign venue name. It will reduce the amount of logic here

Reply:
Agree OOS, probably agree in general. Need to think through the logic/intent here a bit more before I can make that call for certain though.